Doc/library/os.rst

```diff
@@ -5153,14 +5153,21 @@ Miscellaneous System Information
    .. availability:: Unix.
 
 
-.. function:: cpu_count()
+.. function:: cpu_count(*, usable=False)
 
-   Return the number of CPUs in the system. Returns ``None`` if undetermined.
+   Return the number of logical CPUs in the system. Returns ``None`` if
+   undetermined.
 
-   This number is not equivalent to the number of CPUs the current process can
-   use.  The number of usable CPUs can be obtained with
-   ``len(os.sched_getaffinity(0))``
+   If *usable* is true, return the number of logical CPUs the current process
+   can use.  If the :func:`sched_getaffinity` function is available, return
```

Comment:
```suggestion
   can use. If the :func:`sched_getaffinity` function is available, return
```

Reply:
In English, sentences are separated by 2 spaces. My text editor implements English grammar :-)